test/api.js

```diff
@@ -33,6 +33,25 @@ test('alex()', function(t) {
   )
 })
 
+test('alex() with an allow array', function(t) {
+  let {messages} = alex(
```

Comment:
Could you use `var messages = alex(...).messages` here (and on L45)? Rest of the codebase uses ES5.